### PR TITLE
Storage system first iteration

### PR DESCRIPTION
**Pull request by PutinCheese**
1.Add a empty gameObject and give it the storage.cs and a collider.
2.Place it where you want the storage slot
3.Tadaa

### CODE REVIEW

File: behind-the-tankard/Assets/Scripts/Interactable/Categories/Storage.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            PickUpItem item = _inventory.Items.First();
+            _inventory.ExchangeItem(item, player);  
+        }
+    }
+
+    public override void InteractWith(GameObject player, Interactable item)
+    {       
+            player.GetComponent<IInventorist>().GetInventory().ExchangeItem(item.GetComponent<PickUpItem>(), gameObject);
+            
+    }
```
Comment (Justnuked) [inline behind-the-tankard/Assets/Scripts/Interactable/Categories/Storage.cs]: Whitespace and indentation

File: behind-the-tankard/Assets/Scripts/Interactable/Categories/Nourishment.cs [hunk 2]
```diff
@@ -15,6 +15,9 @@ public class Nourishment : PickUpItem
 
     public override void InteractWith(GameObject player, Interactable focus)
     {
+
```
Comment (Justnuked) [inline behind-the-tankard/Assets/Scripts/Interactable/Categories/Nourishment.cs]: White space

File: behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs [hunk 3]
```diff
@@ -198,6 +198,7 @@ private void PlayerInput()
     {
         if (Input.GetButtonDown("Fire1"))
         {
+            
             if (hasFocus && _inventory.HasInventory())
             {
```
Comment (Justnuked) [inline behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs]: White space

File: behind-the-tankard/Assets/Scripts/Interactable/Categories/Storage.cs (hunk 1 above)
Comment (PutinCheese) [reply behind-the-tankard/Assets/Scripts/Interactable/Categories/Storage.cs]: oopsie daisie, my bad

File: behind-the-tankard/Assets/Scripts/Interactable/Categories/Storage.cs [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        {
+            PickUpItem item = _inventory.Items.First();
+            _inventory.ExchangeItem(item, player);  
+        }
+    }
+
+    public override void InteractWith(GameObject player, Interactable item)
+    {       
+            player.GetComponent<IInventorist>().GetInventory().ExchangeItem(item.GetComponent<PickUpItem>(), gameObject);
+
```
Comment (TJ-Smits) [inline behind-the-tankard/Assets/Scripts/Interactable/Categories/Storage.cs]: white space is unnecessary

File: behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs (hunk 3 above)
Comment (TJ-Smits) [reply behind-the-tankard/Assets/Scripts/Interactable/PlayerMovement.cs]: Whitespace can be removed

File: behind-the-tankard/Assets/Scripts/Interactable/Categories/Nourishment.cs (hunk 2 above)
Comment (TJ-Smits) [reply behind-the-tankard/Assets/Scripts/Interactable/Categories/Nourishment.cs]: White space can be removed

File: behind-the-tankard/Assets/Scripts/Interactable/Categories/Storage.cs [hunk 5]
```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class Storage : Interactable, IInventorist
+{
+    public int capacity;
```
Comment (YWAMWeessies) [inline behind-the-tankard/Assets/Scripts/Interactable/Categories/Storage.cs]: This variable is never used. Should it not be removed or have an active meaning in the code

**Comment by TJ-Smits** [comment]
me like. approved

**Pull request merged**